.circleci/config.yml

```diff
@@ -107,6 +107,23 @@ jobs:
               --source-disk-zone="$GOOGLE_COMPUTE_ZONE" \
               --storage-location=us
 
+  publish-libs:
+    docker:
+      - image: circleci/node:14.17.5
+    resource_class: xlarge
```

Comment:
Anyone have any ideas why I was seeing us run out of heap memory, even with large instances?

https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol/25036/workflows/444ccd56-e764-44ea-ac4c-cd872b9fc761/jobs/298665

Reply:
Ok, ty for the confirmation!

I'll move forward with this PR and we'll fix that via INF-182.

Reply:
I don't think we know of a fix though, I doubt this will be fixed anytime soon